Composer/packages/tools/language-servers/language-generation/src/utils.ts

```diff
@@ -76,13 +76,13 @@ export function convertDiagnostics(lgDiags: LGDiagnostic[] = [], document: TextD
 
 export function textFromTemplate(template: Template): string {
   let text = '';
-  if (template.Name && (template.Body !== null && template.Body !== undefined)) {
-    text += `# ${template.Name.trim()}`;
-    if (template.Parameters && template.Parameters.length > 0) {
-      text += '(' + template.Parameters.join(', ') + ')';
+  if (template.name && (template.body !== null && template.body !== undefined)) {
+    text += `# ${template.name.trim()}`;
+    if (template.parameters && template.parameters.length > 0) {
```

Comment:
```suggestion
    if (template.parameters && template.parameters.length) {
```

Consider always coalesce `parameters` with `[]`, this will reduce code and less bug prone.